src/Umbraco.Core/Services/MediaEditingService.cs

```diff
@@ -101,4 +108,6 @@ private ContentEditingOperationStatus Save(IMedia media, int userId)
             return ContentEditingOperationStatus.Unknown;
         }
     }
+
+    private async Task<int> GetUserIdAsync(Guid userKey) => await _userIdKeyResolver.GetAsync(userKey) ?? Constants.Security.SuperUserId;
```

Comment:
Everywhere else this seems to be such that the userKey is inputed to the service layer. Maybe we should align.

No matter what, I dont like the idea about automatically using SuperUser

Reply:
We agreed on huddle that we on a separate PR will fix this, so _userIdKeyResolver.GetAsync throws instead of being nullable